scripts/deploy-helm.sh

```diff
@@ -211,6 +225,7 @@ elif [[ $DEPLOYMENT_TYPE == logs ]]; then
   API_TOKEN_SCOPES=('"logs.ingest"')
 elif [[ $DEPLOYMENT_TYPE == metrics ]]; then
   info "Deploying $DEPLOYMENT_TYPE ingest"
+  # shellcheck disable=SC2034  # Unused variables left for readability
```

Comment:
https://github.com/koalaman/shellcheck/wiki/Directive#disable

Reply:
it should stay as it is because is unused only if script no go into this if but in other cases we used it. in other hand if we add this to config then we ignore all unused errors